index.ts

```diff
@@ -349,6 +350,8 @@ export const isEmptyRepoRoot = (): boolean => isRepoHome() && !exists('link[rel=
 
 export const isEmptyRepo = (): boolean => exists('[aria-label="Cannot fork because repository is empty."]');
 
+export const isArchivedRepo = (): boolean => isRepo() && $('#repository-container-header .Label')?.textContent!.endsWith('archive');
```

Comment:
`?.` will silence any errors here, but I think we shouldn't throw errors when using detections. Thoughts?

Reply:
Yes we should.

Nothing to do with is PR but my last one when it hung instead of throwing.

Reply:
To clarify: should throw or should silence?

Reply:
Throw